airflow/config_templates/default_airflow.cfg

```diff
@@ -362,6 +362,9 @@ cookie_samesite =
 # Default setting for wrap toggle on DAG code and TI log views.
 default_wrap = False
 
+# Allow the UI to be rendered in a frame
```

Comment:
Hi. Can you add some documentation? I think that a short description in the security.rst file was enough. Very few people will use this feature without documentation.

Reply:
see https://github.com/apache/airflow/pull/6995